.github/workflows/sdk-jobs-automl-standalone-jobs-automl-distributed-classification-task-creditcard-classification-distributed-lightgbm.yml

```diff
@@ -0,0 +1,77 @@
+# This code is autogenerated.
+# Code is generated by running custom script: python3 readme.py
+# Any manual changes to this file may cause incorrect behavior.
+# Any manual changes will be overwritten if the code is regenerated.
```

Comment:
Can be dangerous to edit these files, in my experience, as these comments indicate. Did you check with the github actions owners to make sure this is ok?

Reply:
@EricWrightAtWork This is autogenerated by readme.py. Who are the github action owners with whom I can consult?